descwl_shear_sims/lsst_bits.py

```diff
@@ -6,6 +6,11 @@
 COSMIC_RAY = np.int64(2**3)
 EDGE = np.int64(2**4)
 
+# these are not official values, as far as I know
+# the stack doesn't mark such things (ES 2020-02-18)
+STAR = np.int64(2**10)
+BLEED = np.int64(2**11)
```

Comment:
Do we want to go higher here so things don't break later?

Reply:
We will definitely be changing these so it doesn't matter too much.

It looks like the stack masks are 32 bit so we could use 29 and 30